src/lib/ams_msg.c

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2023 Intel Corporation
+ *
+ * Author: Ievgen Ganakov <ievgen.ganakov@intel.com>
+ */
+
+#include <sof/lib/ams_msg.h>
+
+/* Key-phrase detected AMS message uuid: 80a11122-b36c-11ed-afa1-0242ac120002*/
+DECLARE_SOF_RT_UUID("ams_kpd_msg", ams_kpd_msg_uuid, 0x80a11122, 0xb36c, 0x11ed,
+		    0xaf, 0xa1, 0x02, 0x42, 0xac, 0x12, 0x00, 0x02);
```

Comment:
we're still discussing whether we want to collect UUIDs, right? We probably won't get too many AMS users but still I don't see a value in having to change a common file every time you add a new user.